vowpalwabbit/core/src/vw.cc

```diff
@@ -751,6 +751,11 @@ void VW::setup_example(VW::workspace& all, VW::example* ae)
   // Set the interactions for this example to the global set.
   ae->interactions = &all.interactions;
   ae->extent_interactions = &all.extent_interactions;
+
+  if (all.calculate_feature_space_hash)
```

Comment:
isnt this LAS specific since it's invalidated in [one_pass_svd_impl.cc](https://github.com/VowpalWabbit/vowpal_wabbit/pull/4509/files#diff-a8aeef01b1fd9b1fecf12e7021c61ffd8208eb180cd97cba3643a966668d09cc) line 58

Reply:
that's LAS's internal cache of example hashes that is has seen